bftengine/src/bftengine/messages/ViewChangeMsg.cpp

```diff
@@ -32,6 +32,8 @@ ViewChangeMsg::ViewChangeMsg(ReplicaId srcReplicaId,
   b()->genReplicaId = srcReplicaId;
   b()->newView = newView;
   b()->lastStable = lastStableSeq;
+  b()->numberOfComplaints = 0;
```

Comment:
Nit: Since the values are predefined, we may init them in the struct declaration.
Please don't change unless you find it useful.

Reply:
I think it is a good point to highlight. 
I see in the code that we do not need to set anything to 0, because we memset the chunk of memory to 0 [MessageBase.cpp:86](https://github.com/vmware/concord-bft/blob/1b877d35de2af14335a1099878b8f155d311b645/bftengine/src/bftengine/messages/MessageBase.cpp#L86.). Still, I think stating it explicitly is good for clarity.
On the other hand since we malloc the chunk in MessageBase, we don't call any constructors on it, so unfortunately I think setting it to 0 in the struct declaration won't have effect in our use case.

Reply:
Good point!